src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -701,8 +701,8 @@ public synchronized void dropEntity(String entity) throws DatabaseEngineExceptio
      */
     @Override
     public synchronized void dropEntity(final DbEntity entity) throws DatabaseEngineException {
-        dropSequences(entity);
```

Comment:
Is this change in the order intentional ?  If yes, please leave a comment indicating why sequences have to be dropped after tables.

Reply:
It's intentional, I'll add a comment.
Most of the engines don't even use sequences and `dropSequences` is a no-op for them.
DB2 and Oracle use them, but only as the value for the autoinc column in INSERT prepared statements.

CockroachDB uses autoinc like PostgreSQL, but the default is to translate SERIAL as a unique  numeric ID, to have the autoinc functionality a sequence had to be created; however, in this case, there is a dependency on that sequence from the table that uses it, preventing the sequence from being dropped.